src/main/java/uk/gov/hmcts/divorce/ccd/NoFaultDivorce.java

```diff
@@ -20,7 +20,7 @@
 @Component
 public class NoFaultDivorce implements CCDConfig<CaseData, State, UserRole> {
 
-    public static final String CASE_TYPE = "NO_FAULT_DIVORCE16";
+    public static final String CASE_TYPE = "NO_FAULT_DIVORCE17";
```

Comment:
you don't need to bump the ID here, it's only when you make a backwards incompatible change like changing a field type